spec/jobs/perform_search_job_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PerformSearchJob, type: :job do
+  include ActionView::TestCase::Behavior
+  include ActionCable::TestHelper
+  include ApplicationHelper
+
+  let(:user) { create(:user) }
+
+  describe '#perform', vcr: 'jobs/perform_search_job' do
+    let(:query_id) { SecureRandom.uuid }
+
+    it 'successfully executes' do
+      described_class.perform_now('foo', user, query_id)
+    end
+
+    it 'broadcasts the search results' do
+      ActiveJob::Base.queue_adapter = :test
```

Comment:
`ActiveJob::Base.queue_adapter = :test` should be present in the `test.rb` file. This might be redundant

Reply:
Done!